### PR TITLE
Feature/set api

### PR DESCRIPTION
**Pull request by AlucardSanin**
> Create a new game with the name of your choice by using the Leaderboard API.
> Save the ID of your game that will be returned by API.
> Implement the "Refresh" button that will get all scores for a game created by you from the API.
> Implement the form "Submit" button that will save a score for a game created by you (sending data to the API).
> Use arrow functions instead of the function keyword.
> Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/APIset.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (ahmetbozaci) [inline src/APIset.js]: - [ ] Kindly don't disable linter errors. If giving error please fix it.

File: src/index.html [hunk 2]
```diff
@@ -10,13 +10,18 @@ <h1>Leaderboard</h1>
...
     </div>
     <div>
         <h2>Add your score</h2>
-        <input type="text" name="user" placeholder="Your name">
-        <input type="text" name="score" placeholder="Your score">
+        <form>
+            <label for="user">User:</label>
+            <input type="text" id="user" placeholder="Your name" required>
+            <label for="score">Score:</label>
+            <input type="text" id="score" placeholder="Your score" required>
```
Comment (ahmetbozaci) [inline src/index.html]: OPTIONAL
- [ ] I can add text as a score kindly change type to number

File: src/index.js [hunk 3]
```diff
@@ -1,12 +1,35 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (ahmetbozaci) [inline src/index.js]: Kindly don't disable linter errors. If giving error please fix it.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @AlucardSanin  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- I can add scores and with click the refresh button ı can see the scores that ı added.✔️
- You used arrow functions.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @AlucardSanin, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**